docs/subgraph-manifest.md

```diff
@@ -64,6 +64,7 @@ The `mapping` field may be one of the following supported mapping manifests:
 | --- | --- | --- |
 | **event** | *String* | An identifier for an event that will be handled in the mapping script. For Ethereum contracts, this must be the full event signature to distinguish from events that may share the same name. No alias types can be used. For example, uint will not work, uint256 must be used.|
 | **handler** | *String* | The name of an exported function in the mapping script that should handle the specified event. |
+| **topic0** | optional *String* | A `0x` prefixed hex string. If provided, logs whose topic 0 is equal to this value will be processed by the given handler. By default, this is equal to the hash of the event signature. This is useful for processing solidity anonymous events. |
```

Comment:
solidity anonymous events -> anonymous events in Solidity

Reply:
I'd also mention that if topic0 is provided, the subgraph will index events with matching to topic0 only, _instead_ of all events with the given signature.

Reply:
I added in Jannis' comments, and some of my own for what I think would make sense to a Solidity developer

A `0x` prefixed hex string. If provided, logs whose topic0 is equal to this value will be processed by the given handler. When topic0 is provided, the subgraph will _only_ match the topic0 hash, and it will not match on the hash of the event signature. This is useful for processing anonymous events in Solidity, which can have their topic0 set to anything.  By default, topic0 is equal to the hash of the event signature.

Reply:
👍 Just one thing: Let's talk about `events` rather than `logs` wherever we can. It makes our language more consistent.

Reply:
Thanks for the suggestions! I've updated these docs.